changelogs/fragments/7810.yml

```diff
@@ -0,0 +1,2 @@
+feat:
+- Add OpenSearch PPL autocomplete to discover 2.0 with query enhancements ([#7810](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7810))
```

Comment:
np. there are two changelogs but non of them is for #7932?

Reply:
Just updated it, added more details :)